cmd/util/util.go

```diff
@@ -165,3 +167,89 @@ func PromptUserForAccept(r io.Reader) bool {
 		return false
 	}
 }
+
+func MaybePrintKubectlDownloadMsg() {
+	if !viper.GetBool(config.WantKubectlDownloadMsg) {
+		return
+	}
+
+	// darwin, linux and windows are only available k8s url options
+	//   all possible runtime.GOOS options: "android darwin dragonfly freebsd linux nacl netbsd openbsd plan9 solaris windows "
```

Comment:
I think that this isn't true - I'm pretty sure kubectl arm and 386 binaries are also uploaded.  
I'd be fine just injecting runtime.GOOS into the format string.  

I also think its fine to just leave

```
curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/%s/bin/%s/%s/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
```
as the install instructions regardless of OS to skip this entire block.  The windows specific instruction isn't immediately runnable either so it doesn't really provide anything additional (and some windows users might have curl installed on windows anyways)